docs/release-notes/8.16.asciidoc

```diff
@@ -0,0 +1,115 @@
+* Adds a {kib} advanced setting `securitySolution:maxUnassociatedNotes`, which allows you to set the maximum number of notes that can be attached to alerts and events ({kibana-pull}194947[#194947]). 
+* Adds the `IS` operator as an option when configuring a Windows signature blocklist entry ({kibana-pull}190515[#190515]). 
+* Allows you to disable the {elastic-defend} hardware call stacks feature ({kibana-pull}190553[#190553]). 
+* Improves network previews in the alert details flyout ({kibana-pull}190560[#190560]). 
+* Enhances {elastic-defend} by improving the `call_stack_final_user_module` attribution where potential `proxy_call` modules are encountered during Windows call stack analysis.
+* Adds new fields to {elastic-defend} API events to improve context for the triage of Behavior Alerts. The new `call_stack_final_user_module` fields are `allocation_private_bytes`, `protection`, `protection_provenance_path`, and `reason`.
+* Adds a new {elastic-defend} API event for https://learn.microsoft.com/en-us/windows/win32/api/ioapiset/nf-ioapiset-deviceiocontrol[`DeviceIoControl`] calls to support the detection of driver abuse. This feature is only supported on Windows 11 Desktop versions.
+* Ensures security artifacts are updated when the {elastic-defend} service starts.
+* Improves error messages that are returned when {elastic-defend} receives invalid or unsupported cryptographic keys from the {elastic-defend} policy. 
+* Ensures that {elastic-defend} tells {fleet} that it's `orphaned` if the connection between {elastic-defend} and {agent} stops for an extended period of time. {fleet} uses this information to provide you with additional troubleshooting context.
+* Adds SOCKS5 proxy support to {elastic-defend}'s {ls} output.
+* Ensures that on Windows, {elastic-defend} uses https://www.elastic.co/security-labs/finding-truth-in-the-shadows[Intel CET and AMD Shadow Stacks] to collect call stacks, where supported. This improves performance and enables detection of certain defense evasions.
+* Restore {elastic-defend}'s support for Windows Server 2012, which was removed in 8.13.0.
+* Improves {elastic-defend}'s caching to reduce memory usage on Windows.
+* Enhances {elastic-defend} by reducing the size of process events, which reduces excessive process ancestry entries and shortens the entity ID.
+* Improves the reliability and system resource usage of {elastic-defend}'s Windows network driver.
+
+[discrete]
+[[bug-fixes-8.16.0]]
+==== Bug fixes
+
+* Prevents an empty warning message from appearing for rule executions ({kibana-pull}186096[#186096]). 
+* Fixes an error that could occur during rule execution when the source index had a non-ECS-compliant text field ({kibana-pull}187673[#187673]). 
+* Removes unnecessary empty space below the title of the Open Timeline modal ({kibana-pull}188837[#188837]). 
+* Added a tag that was missing from an FTR suite ({kibana-pull}189661[#189661]). 
```

Comment:
This seems more dev related, does it need to be in public docs?

Reply:
Nope! I'll remove it.

```suggestion
```